### PR TITLE
Inactivity Penalty no more given if stake less than min stake

### LINKED ISSUE #397: Should we render penalty if stake is lesser than min stake?
Should lock penalty and inactivity penalty be applied if stake < minstake?

### PR DESCRIPTION
**Pull request by SamAg19**
fixes #397

### CODE REVIEW

File: test/StakeManager.js [hunk 1]
```diff
@@ -1416,5 +1416,35 @@ describe('StakeManager', function () {
...
+      await mineToNextEpoch();
+
+      epoch = await getEpoch();
+      await stakeManager.setStakerStake(epoch, stakerId, 1, tokenAmount('500'));
+
+      const epochsJumped = GRACE_PERIOD + 4;
+      for (let i = 0; i < epochsJumped; i++) {
+        await mineToNextEpoch();
+      }
+
```
Comment (abhishekvispute) [inline test/StakeManager.js:1436]: hey, once grace period is over, inactivity penalties are applied only when you commit or unstake
this test would have hold true without this pr as well.
right ?
I think we should do unstable here

Comment (SamAg19) [reply test/StakeManager.js:1436]: I have unstaked at L1445

Comment (abhishekvispute) [reply test/StakeManager.js:1436]: cool

**Review by abhishekvispute — APPROVED** [review]

**Comment by hrishikeshio** [comment]
I don't think it fixes the issue.

It only checks the current minstake.

T1
Stake 20000
Minstake 10000

Staker goes down for 1000 epochs and stays down

T2
Stake 20000
Minstake 30000

Staker withdraws without inactivity penalty.

**Comment by SamAg19** [comment]
> I don't think it fixes the issue.
> 
> It only checks the current minstake.
> 
> T1
> Stake 20000
> Minstake 10000
> 
> Staker goes down for 1000 epochs and stays down
> 
> T2
> Stake 20000
> Minstake 30000
> 
> Staker withdraws without inactivity penalty.

How about we give out inactivity penalties to all the stakers just before we change minStake when we call setMinStake function in Parameters? The gas costs will be incurred by us here as the admin can only call this function

**Comment by hrishikeshio** [comment]
Lets discuss in the issue.

**Pull request closed without merging**